### ISSUE TITLE
Explain origin and generation of CSV/ automate procedure

### ISSUE THREAD

**Issue by SArndt-TIB** [tibonto/DFG-Fachsystematik-Ontology#14]
The repo currently does not explain how the csv file is created, e.g. whether it results from an automated processes.

**Comment by andrecastro0o** [comment]
@SArndt-TIB I will be trying to describe the actions I think I need to take in order to create a CSV from the .xlsx files

1. remove rows 1,2 (title, empty)
2. remove empty columns A and F
3. add to header (row 1) "Subject Area" and "Scientific Discipline" in columns D, E
4. repeat 3. for row 57, 137, 169
5. split header `Subject Number` and  `Subject` headers

CSV export: Check "Quote all text cells"

(in progress)

**Comment by andrecastro0o** [comment]
# Bugs in CSV - manual clean up needed

It is likely that the will some bugs in the CSV - mostly `label_formatting` , `label_whitespace` and `annotation_whitespace` errors, that will be detected by ROBOT in [Ontology testing](https://github.com/tibonto/DFG-Fachsystematik-Ontology/actions/workflows/main.yml)  Github Actions.

The recommendation is to **fix these errors in the CSV (the working document)**. I use a plain-text/code editor do those edits, as it allows me to see better the text, and use search and replace. The following commits are examples of these edits https://github.com/tibonto/DFG-Fachsystematik-Ontology/pull/16/commits/6130123dfc9864203a9649b349cee86a4afe90ab https://github.com/tibonto/DFG-Fachsystematik-Ontology/pull/16/commits/f03a411d1287464a16d7ce04ee1bd088c983d107 https://github.com/tibonto/DFG-Fachsystematik-Ontology/pull/16/commits/f7e0bb33fede3f1a5e2e6bb11f8c882027afcc55

Example ROBOT errors:
```
ERROR	label_formatting	https://github.com/tibonto/dfgfo/2.23-10	rdfs:label	Clinical Psychiatry, Psychotherapy, 
ERROR	label_whitespace	https://github.com/tibonto/dfgfo/1.13	rdfs:label	 Art History, Music, Theatre and Media Studies@en
ERROR	label_whitespace	https://github.com/tibonto/dfgfo/1.16-01	rdfs:label	Social and Cultural Anthropology and Ethnology @en
```

**Comment by SArndt-TIB** [comment]
@andrecastro0o I started to add a readme for this on branch 13, I will continue this with your info from this issue. Probably not today, though.

**Comment by andrecastro0o** [comment]
@SArndt-TIB shall we close this issue? If there will be more related about it we can re-open or create a new one

**Comment by andrecastro0o** [comment]
Issue addressed in #16
